src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -136,15 +137,13 @@ protected override void Generate(MigrationOperation operation, IModel model, Mig
                 var columnComment = columnOp.Comment;
                 builder.AppendLine(';');
 
-                var stringTypeMapping = Dependencies.TypeMappingSource.GetMapping(typeof(string));
-
                 builder
                     .Append("COMMENT ON COLUMN ")
                     .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(operation.Name, operation.Schema))
                     .Append('.')
                     .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(columnOp.Name))
                     .Append(" IS ")
-                    .Append(stringTypeMapping.GenerateSqlLiteral(columnComment));
+                    .Append(_stringTypeMapping.GenerateSqlLiteral(columnComment));
```

Comment:
What is difference between resolving a mapping from `Dependencies` in a method and doing the same thing in a constructor? Is it for optimization only?

Reply:
No difference, just to clean up the code and not repeat the mapping lookup...